src/Authorize/Token.php

```diff
@@ -2,9 +2,6 @@
 
 namespace TrueLayer\Authorize;
 
-use DateInterval;
-use DateTime;
```

Comment:
To further @kgdiem change. Do we need these lines at-all?

Reply:
Yes, I think so. 

Otherwise you will get an error `Date[Interval | Time]` does not exist in namespace `TrueLayer\Authorize`

Reply:
I have changed the use of the classes too.
They are now `return new \DateTime($value);`

Reply:
ah okay. Didn't see that, sorry. I guess it's up to @waxim as to which he prefers. 

I'm not familiar as to how scope resolution is handled in this situation, eg which method is more performant.

I personally rather a single import than continually adding `\` whenever the class is used.

Reply:
I think @waxim is on the same page as you. I'm used to seeing backslashes on built in classes but either way it doesn't really matter.

Reply:
ya, same. Not important to me, I just want it to work. Thanks for your contributions (as a fellow contributor)